fw/vhost.c

```diff
@@ -1345,8 +1347,16 @@ tfw_location_new(TfwVhost *vhost, tfw_match_t op, const char *arg, size_t len)
 		return NULL;
 	vhost->loc_sz++;
 
-	if (tfw_frang_cfg_inherit(loc->frang_cfg, vhost->loc_dflt->frang_cfg))
-		return NULL;
+	if (strncasecmp(vhost->name.data, TFW_VH_DFT_NAME, vhost->name.len)) {
```

Comment:
It seems that it is totally wrong or I don't understand the idea.
frang_limits {
     http_body_len 30;
}
 frang_limits {
      http_body_len 90;
 }

 location prefix "/static/" {
     frang_limits {
          http_body_len 270;
     }
 }

What http_body_len we should have here? I think that 270. If there is no frang_limits in location we should have 90, but I print that we have default value 1073741824. Please add test how it works.